docs/README.md

```diff
@@ -41,7 +41,7 @@ Before contributing, please review the [Documentation Contribution Flow](https:/
   ```bash
   $ git push origin <branch-name>
   ```
-* Then on Github, navigate to the [Service-Mesh-Performance-Specification repository](https://github.com/layer5io/service-mesh-performance-specification) and create a pull request from your recently pushed changes!
+* Then on Github, navigate to the [Service-Mesh-Performance repository](https://github.com/layer5io/service-mesh-performance-specification) and create a pull request from your recently pushed changes!
```

Comment:
and this repo too.

Reply:
Good eye. This repo has yet to change names, so the old references were untouched.